auth/lib/utils/totp_util.dart

```diff
@@ -4,7 +4,7 @@ import 'package:otp/otp.dart' as otp;
 import 'package:steam_totp/steam_totp.dart';
 
 String getOTP(Code code) {
-  if (code.issuer.toLowerCase() == 'steam') {
```

Comment:
Hello,
Can you please change this to use `code.type == Type.steam`

And modify following line to handle domain as issuer

https://github.com/ente-io/ente/blob/main/auth/lib/onboarding/view/setup_enter_secret_key_page.dart#L243

```dart
final isStreamCode =
          issuer.toLowerCase() == "steam" || issuer.toLowerCase().contains('steampowered.com');
```

Thanks for the PR :)

Reply:
Thank you for checking my PR! I will change both IFs with `code.type == Type.steam` in [auth/lib/utils/totp_util.dart](https://github.com/ente-io/ente/pull/1809/files/954581093dd751fecf3d823985a8dac00ce74dfc#diff-a2a9b59a01e1ebc767cb4c397fcd346b91686783c9dd0a0cc5aceced09e553b0), update [auth/lib/onboarding/view/setup_enter_secret_key_page.dart#L243](https://github.com/ente-io/ente/blob/main/auth/lib/onboarding/view/setup_enter_secret_key_page.dart#L243), add changes suggested below by @prateekmedia and push a new commit.